test/phone-number-generator.test.ts

```diff
@@ -12,6 +13,7 @@ describe("generatePhoneNumber", () => {
       const phoneNumber = generatePhoneNumber();
       const endTime = Date.now();
       expect(isPhoneNumber(phoneNumber)).toBeTruthy();
+      expect(invalidPhoneNumbersNumbers.includes(phoneNumber)).toBeFalsy();
```

Comment:
I don't think we need to add this check to every test.
Let's create another test for this use cases only